README.md

```diff
@@ -52,8 +52,25 @@ Example of `~/.ungitrc` configuration file to change default port and enable bug
 
 ```json
 {
-	"port": 8080,
-	"bugtracking": true
+  "port": 8080,
+  "bugtracking": true,
+  "gitCommand": "D:\\Apps\\Git\\bin\\git"
```

Comment:
Can we make it explicit that `gitCommand` or `gitPath` being null is a valid config?